Gemfile

```
@@ -0,0 +1,53 @@
+source "https://rubygems.org"
+git_source(:github){|repo| "https://github.com/#{repo}.git"}
+
+ruby "2.7.0"
+
+gem "rails", "~> 6.1.4", ">= 6.1.4.1"
+
+gem "sqlite3", "~> 1.4"
+
+gem "puma", "~> 5.0"
```

```suggestion
gem "rails", "~> 6.1.4", ">= 6.1.4.1"
gem "sqlite3", "~> 1.4"
gem "puma", "~> 5.0"
```
xóa mấy line trống thừa đi, self check file này

chạy `bundle exec rubocop` đi bé